js_modules/dagster-ui/packages/ui-core/src/runs/LogsRowStructuredContent.tsx

```diff
@@ -184,6 +184,7 @@ export const LogsRowStructuredContent = ({node, metadata}: IStructuredContentPro
     case 'AlertFailureEvent':
       return <DefaultContent eventType={eventType} message={node.message} eventIntent="warning" />;
     case 'ResourceInitFailureEvent':
+    case 'RunCanceledEvent':
```

Comment:
probably want to remove the case on line 205

Reply:
Ooof good call, surprised that wasn't a TS error!